src/components/institution/OrganizationBox.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, BoxProps, styled, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetchOrganization } from '../../api/hooks/useFetchOrganization';
+import { DeleteIconButton } from '../../pages/messages/components/DeleteIconButton';
+import { dataTestId } from '../../utils/dataTestIds';
+import { AffiliationSkeleton } from './AffiliationSkeleton';
+import { OrganizationHierarchy } from './OrganizationHierarchy';
+
+export const StyledOrganizationBox = styled(Box)({
+  border: '1px solid',
+  borderRadius: '4px',
+  display: 'flex',
```

Comment:
Lurt med en gap også, i de tilfellene navnet trenger hele bredden
```suggestion
  display: 'flex',
  gap: '0.5rem',
```
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/f2a048b6-b005-44f2-a56d-c70d3fa85256)